Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -4230,6 +4233,9 @@ script:
     - description: The tag of the service group to edit.
       isArray: true
       name: tags
+    - description: The device group in which the service group belongs to.
+      name: device_group
```

Comment:
why add with a _? Looks like the other args are with a - and the param with a _. Seems like a good way to distinguish. 
I would add DEVICE_GROUP_PARAM_NAME and DEVICE_GROUP_ARG_NAME constants to the code

Reply:
_ was part of the design, ill change it to - to align with the rest